dtfs-central-api/src/v1/swagger.js

```diff
@@ -28,11 +29,16 @@ const swaggerDefinition = {
       name: 'User',
       description: 'Get and create Portal (BSS/GEF) users. This is only used in the central API.',
```

Comment:
Legacy but do you know what `This is only used in the central API` mean? Is it that these endpoints are only used in tests?

Reply:
I have no idea